### ISSUE TITLE
How can it be ran on Cifar-100?

### ISSUE THREAD

**Issue by wishforgood** [bamos/densenet.pytorch#13]
I changed the dataset class name in the code from CIFAR10 to CIFAR100 but got serveral error during loss.backward(), like `CUDNN_STATUS_MAPPING_ERROR` or `cublas runtime error the gpu program failed to execute`. So I guess there must be something specified to CIFAR10 in this code. But I can't find it.

**Comment by wishforgood** [comment]
I just forgot to change nClasses.

**Comment by bamos** [comment]
That's a confusing error message for that mistake, glad you fixed it!
